src/object/class/internal.rs

```diff
@@ -112,6 +131,16 @@ impl<T: Class> Callback<bool> for ConstructCallback<T> {
                 let data = info.data();
                 let kernel: fn(CallContext<T>) -> NeonResult<Option<Handle<JsObject>>> =
                     mem::transmute(neon_runtime::class::get_construct_kernel(data.to_raw()));
+                #[cfg(feature = "napi-runtime")]
```

Comment:
This is one of the duplications we can eliminate by having a consistent signature for `convert_panics`.